### PR TITLE
Bump to version 0.26.0

### PR DESCRIPTION
**Pull request by delner**

### CODE REVIEW

File: lib/ddtrace/runtime/cgroup.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+      module_function
+
+      def descriptors(process = 'self')
+        [].tap do |descriptors|
+          begin
+            filepath = "/proc/#{process}/cgroup"
+
+            if File.exist?(filepath)
+              File.open("/proc/#{process}/cgroup").each do |line|
```
Comment (mattsp1290) [inline lib/ddtrace/runtime/cgroup.rb]: This should probably be File.open(filepath) so you only have to change it in one place

File: lib/ddtrace/writer.rb [hunk 2]
```diff
@@ -79,29 +78,20 @@ def send_spans(traces, transport)
...
 
-        # Return if server error occurred.
-        !response.server_error?
+        # Update priority sampler
+        unless priority_sampler.nil? || response.service_rates.nil?
+          priority_sampler.update(response.service_rates)
+        end
       end
+
+      # Return if server error occurred.
```
Comment (mattsp1290) [inline lib/ddtrace/writer.rb]: This feels a little misleading. It looks like we return whether or not a server error was absent. IE if there was a server error we return false and if there was not we will return true.

File: lib/ddtrace/runtime/cgroup.rb (hunk 1 above)
Comment (delner) [reply lib/ddtrace/runtime/cgroup.rb]: Thanks, must have forgot to update line 24. We'll do this in a future PR.

File: lib/ddtrace/writer.rb (hunk 2 above)
Comment (delner) [reply lib/ddtrace/writer.rb]: Yeah maybe the comment isn't as accurate, but the return value looks correct.

**Pull request merged**